src/explore-education-statistics-admin/src/pages/release/datablocks/components/DataBlockPageReadOnlyTabs.tsx

```diff
@@ -31,10 +31,7 @@ const testId = (dataBlock: ReleaseDataBlock) =>
 
 const DataBlockPageReadOnlyTabs = ({ releaseId, dataBlock }: Props) => {
   const { value: model, isLoading } = useAsyncRetry<Model>(async () => {
-    const query: ReleaseTableDataQuery = {
-      ...dataBlock.query,
-      includeGeoJson: dataBlock.charts.some(chart => chart.type === 'map'),
-    };
+    const query: ReleaseTableDataQuery = { ...dataBlock.query };
```

Comment:
This doesn't need to be spread now nothing's being added to it, you can just pass `dataBlock.query` directly into `getTableData`.